components/Explore/CardLabel.tsx

```diff
@@ -1,30 +1,37 @@
 import { MaterialCommunityIcons } from "@expo/vector-icons";
 import { View, Text } from "react-native";
-import PropTypes from "prop-types";
+import { colors } from "@/theme/colors";
+
+export interface CardLabelProps {
+  title: string;
+  icon: keyof typeof MaterialCommunityIcons.glyphMap;
+  color?: string;
+  font?: string;
+}
 
 /**
  * This component is used to display a label in a course card.
  * @param title - The text of the label.
  * @param icon - The icon of the label.
  * @param color - The color of the label and icon.
- * @param time - Boolean value to determine to render the time label.
+ * @param font - Styling of font, such as size and weight.
  * @returns {JSX.Element} - Returns a JSX element.
  */
-const CardLabel = ({ title, icon, color = "gray" }) => {
+export const CardLabel = ({
+  title,
+  icon,
+  color = colors.textCaptionGrayscale,
+  font = "font-semibold text-s",
```

Comment:
```suggestion
  font = "text-body-regular",
```

...or whatever base preset you want to use here